build.gradle.kts

```diff
@@ -8,28 +8,19 @@ import org.flywaydb.gradle.task.FlywayMigrateTask
 import org.jooq.util.jaxb.Jdbc
 import java.sql.DriverManager
 
+val logback_version: String by project
+val ktor_version: String by project
+val kotlin_version: String by project
+val jackson_version: String by project
+val hickarcp_version: String by project
+val kotlin_logging_version: String by project
+val postgresql_version: String by project
 
 
-val ktor_version = "0.9.5"
-val jackson_version = "2.9.4"
-
-buildscript {
-    repositories {
-        gradlePluginPortal()
-        mavenCentral()
-        maven {
-            url = uri("https://plugins.gradle.org/m2/")
-        }
-    }
-    dependencies {
-        classpath("org.postgresql:postgresql:42.2.5")
```

Comment:
This is actually required for the migration to work. Did you test it?

```
./gradlew startPostgres
./gradlew migrateDatabase
```

Reply:
Yup, it fails
```
› ./gradlew migrateDatabase
> Task :createDatabase FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':createDatabase'.
> java.sql.SQLException: No suitable driver found for jdbc:postgresql://localhost:5432/
```

Reply:
Hm, I wanted to remove the `buildscript` section since it defines repositories and dependencies yet again but failed.